WookieeMatt/Essence20
# Adding basic support for Drag Ruler

Closes https://github.com/WookieeMatt/Essence20/issues/590

##### In this PR
- Adding some basic support for the [Drag Ruler](https://foundryvtt.com/packages/drag-ruler/) module
- For now only using ground movement
- Moving `getNumActions()` to `util.mjs`
- I didn't really see actor data within the `token` object, so i resorted to `game.actors.get(token.document.actorId)`. Maybe there's a better way?

##### Testing
- Install Drag Ruler
- Create an actor with no origin (which means they have no movement). Place a token and drag them around. The line coloring should be red.
- Give the actor an origin with some ground movement. Dragging them around should now show green for standard movement, yellow for sprinting (ground x 2), and red beyond that.
